### DISCUSSION TITLE
Passing the input label to yup error messages

### DISCUSSION THREAD

**Discussion by BennyAlex** [lookfirst/mui-rff#1164] [Ideas]
Hi,

currently I can only access the ${path} placeholder in yup error messages which is the same as the field name.

The problem is, that the name of an input is hardcoded and should not be localized.
This results in half translated error messages.

![image](https://github.com/lookfirst/mui-rff/assets/17402724/9d0780d8-b7c2-44c6-9b72-420301e5ae51)
![image](https://github.com/lookfirst/mui-rff/assets/17402724/be80a968-62f6-4ad8-a1fd-4d62f55f5b52)

I already checked if there is an ${label} placeholder but its undefined.

@lookfirst 
Could you pass the label instead of the name to the path if its set or having a another label placeholder?

Have a good day!

**Comment by lookfirst** [comment]
Hi there, this code is in validation.ts. It is entirely possible for you to write your own makeValidation function that does exactly what you want? Happy to accept a PR too.

**Reply by BennyAlex** [reply]
Hey it looks like it is problem with yup as far as I can tell.
The yup error message only has the path, which is the key of the object in the schema. I dont think there is a way to get the correct label then. Or do you have any idea? How do you handle translations for yup errors?
